iceberg/src/main/scala/org/apache/iceberg/spark/functions/bucket.scala

```diff
@@ -93,4 +94,32 @@ object GpuBucketExpression {
       case _ => false
     }
   }
+
+  def tagExprForGpu(meta: ExprMeta[StaticInvoke]): Unit = {
+    require(meta.childExprs.length == 2,
+      s"BucketFunction should have exactly two arguments, got ${meta.childExprs.length}")
+    val exprCls = meta.wrapped.staticObject
+    if (exprCls != classOf[BucketFunction.BucketInt] &&
+      exprCls != classOf[BucketFunction.BucketLong]) {
+      meta.willNotWorkOnGpu(s"only BucketFunction.BucketInt and BucketFunction.BucketLong " +
+        s"are supported")
```

Comment:
Messages like this is more useful if the show the actual value, not just the expected value. 
```suggestion
      meta.willNotWorkOnGpu(s"only BucketFunction.BucketInt and BucketFunction.BucketLong " +
        s"are supported. Actual: ${exprCls}")
```
You did this for dataType